internal/analytics/snapshot_test.go

```
@@ -0,0 +1,202 @@
+	require.NoError(t, err)
+	defer snap.Close()
+
+	// All ten schema tables must be present in the snapshot.
+	wantTables := []string{
+		"sessions", "events", "authorship_ledger", "metacognition_logs",
+		"agent_spans", "feed_cache", "coaching_state", "cost_state",
+		"notifications", "schema_meta",
+	}
+	for _, tbl := range wantTables {
+		var name string
+		row := snap.QueryRow(
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).QueryRow must not be called. use (*database/sql.DB).QueryRowContext

(noctx)